README.md

```diff
@@ -17,6 +17,7 @@ requires `libudev-dev`
 usbmon [-n][-c]
   -n do not monitor events
   -c collectd exec plugin mode
+  -t=TV_SEC set time interval for monitoring default:10
```

Comment:
please do not put TV_SEC like this, make it to a "seconds" or something human can read

Reply:
i can change it to -t=SECONDS or (INTERVAL) since most unix commands have argument to option in UPPERCASE.